javascript/ql/src/semmle/javascript/security/dataflow/DOM.qll

```diff
@@ -211,7 +211,7 @@ private class WindowNameAccess extends RemoteFlowSource {
     this = DataFlow::globalObjectRef().getAPropertyRead("name")
     or
     // Reference to `name` on a container that does not assign to it.
-    this.accessesGlobal("name") and
+    this.asExpr().(GlobalVarAccess).getName() = "name" and
```

Comment:
`this = DataFlow::globalVarRef("name")` would catch a few things that aren't `GlobalVarAccess`es.

Reply:
(And would subsume the first disjunct.)

Reply:
I believe that's handled by the above disjunct.

Reply:
Almost; cf Nodes.qll.

Reply:
Oh, wait, no, entirely. Ignore me.